static/app/components/codeSnippet.tsx

```diff
@@ -13,6 +13,12 @@ interface CodeSnippetProps {
   language: string;
   className?: string;
   dark?: boolean;
+  /**
+   * Makes the text of the element and its sub-elements not selectable.
+   * Userful when loading parts of a code snippet, and
+   * we wish to avoid users copying them manually.
+   */
+  disabledUserSelection?: boolean;
```

Comment:
let's call this `disableUserSelection`, without the `d`

Reply:
done